pydantic_ai_slim/pydantic_ai/realtime/google.py

```
@@ -0,0 +1,652 @@
+            # `receive()` returned normally → the turn ended; loop for the next one.
+
+    async def _try_reconnect(self) -> bool:
+        """Re-dial with exponential backoff, resuming from the latest handle; return whether it worked."""
+        assert self._dial is not None and self._reconnect is not None
+        return await reconnect_with_backoff(self._reconnect, self._attempt_reconnect)
+
+    async def _attempt_reconnect(self) -> bool:
+        assert self._dial is not None
+        try:
+            self._session = await self._dial(self._resumption_handle)
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)